code/modules/research/designs/misc_designs.dm

```diff
@@ -614,3 +614,27 @@
 	build_path = /obj/item/circuitboard/computer/sat_control
 	category = list("Computer Boards")
 	departmental_flags = DEPARTMENTAL_FLAG_ENGINEERING
+
+
+
+/////////////////////////////////////////
+////////////Tackle Gloves////////////////
```

Comment:
These designs are not included in any tech node, not that it matters that much.

Reply:
Yeah, should i remove these from the code or just leave them in for whenever we get the BEPIS((hopefully someday))